static/app/components/searchSyntax/parser.tsx

```diff
@@ -697,17 +697,14 @@ export class TokenConverter {
   });
 
   tokenValueText = (value: string, quoted: boolean) => {
-    let contains: boolean | undefined;
-    if (this.config.parseWildcardsCheckIsEnabled) {
-      contains = value.startsWith('*') && value.endsWith('*');
-    }
+    const isContains = value.startsWith('*') && value.endsWith('*');
 
     return {
       ...this.defaultTokenFields,
       type: Token.VALUE_TEXT as const,
       value,
       quoted,
-      ...(typeof contains === 'boolean' ? {contains} : {}),
+      ...(this.config.parseWildcardsCheckIsEnabled ? {contains: isContains} : {}),
```

Comment:
Is there any reason we need this flag at all?

Reply:
Wanted to be able to control the whole feature work from top to bottom with a feature flag, so this check allows us to conditionally add this field.

Reply:
I think having a flag in here is probably unnecessary.

Reply:
We can just always include this field, since it's such a simple check

Reply:
Alright if I add that in a follow, will require something like 700+ changes for test fixtures

Reply:
I think that's OK. This PR is JUST adding this functionality right?

Reply:
Exactly, the flag is currently set to false everywhere so it won't affect anything if merged.